app/helpers/format_helper.rb

```diff
@@ -176,20 +176,20 @@ def selected_scheduled?(schedule)
     (schedule == @selected_schedule) ? 'Yes' : 'No'
   end
 
-  def markdown(text)
+  def markdown(text, allow_html=false)
     return '' if text.nil?
 
     options = {
       autolink: true,
       space_after_headers: true,
       no_intra_emphasis: true
     }
-    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML.new(escape_html: true), options)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML.new(escape_html: !allow_html), options)
```

Comment:
I wrote it this way, because I think it's more clear when call it. 'allow' to render may be more clear than  'escape' to prevent rendering. I'm making the changes.